src/to_curl.rs

```diff
@@ -118,6 +118,10 @@ pub fn translate(args: Cli) -> Result<Command> {
         }
     }
 
+    if args.curl && args.follow && args.method.as_ref().map(|m| m != &Method::GET).unwrap_or(false) {
```

Comment:
Can we use the `matches!()` macro to simplify things a little bit?

```rust
if args.follow && !matches!(args.method, Some(Method::GET) | None) { /* ... */ }
```

We can also skip checking `args.curl` since `to_curl::print_curl_translation` is only called when `args.curl`  is true.

Reply:
Sure, changed in the latest commit. Should I squeash it ?